src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -1,8 +1,27 @@
 package frc.robot.subsystems;
 
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants.Ports;
+
+import com.revrobotics.CANEncoder;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
 
 public class Intake extends SubsystemBase {
-    // TODO: Create empty intake methods (e.g. move up, move down, run rollers, etc.)
-    // TODO: start implementing methods
+    
+    private final CANSparkMax m_rollerMotor = new CANSparkMax(Ports.ROLLER_MOTOR_PORT, MotorType.kBrushless); //may be actually brushed not sure rn
+    private final CANSparkMax m_intakeMechanismMotor = new CANSparkMax(Ports.INTAKE_MECHANISM_MOTOR_PORT, MotorType.kBrushless); //^^f
+
+    public void moveUp(final int degrees) { 
+    }
+
+    public void moveDown(final int degrees) {
+        
+    }
+
+    public void runRollers(final int rotations) {
+
+    }
+
```

Comment:
Really, in competition, you don't want to tell the robot to move the rollers in a particular direction. Actually, what you want is "go to ground level", or "go to stored level" or "go to scoring level". Therefore, I propose you add a method:

```
moveToPosition(IntakePosition position) {

}
```
where IntakePosition is an enum.

Reply:
Well run rollers isn't for moving up and down. I figured it would be an on/off thing, that controls the actual rollers themselves and whether they are actively intaking power cells

Reply:
but I will make a numerator for what you described